docs/api/ref/DesktopAgent.md

```diff
@@ -165,7 +165,7 @@ Find all the avalable intents for a particular context.
 `findIntentsByContext` is effectively granting programmatic access to the Desktop Agent's resolver.
 A promise resolving to all the intents, their metadata and metadata about the apps that registered as handlers is returned, based on the context types the intents have registered.
 
- If the resolution fails, the promise will return an `Error` with a string from the [`ResolveError`](ResolveError) enumeration.
+If the resolution fails, the promise will return an `Error` with a string from the [`ResolveError`](ResolveError) enumeration.
```

Comment:
Shouldn't this be changed from:

[`ResolveError`](ResolveError)

to:

[`ResolveError`](Errors#resolveerror)

Reply:
yes, the link should be to `Errors#resolveerror`, will fix